### DISCUSSION TITLE
Using files on local file system for execution of code

### DISCUSSION THREAD

**Discussion by rohan-uiuc** [ricklamers/gpt-code-ui#7] [Q&A]
Hi,

I have a question about the ability to upload files. Currently, when you upload a file, it is added to the context. This is great for files that you want to use as input for the code, but it doesn't help in cases where you want to use a large file for execution purposes.

For example, I'm working on a project where I need to use a large dataset to train a machine learning model. I can't upload the dataset to the context because it's too large. I would like to be able to create a workspace and add the dataset to the workspace. This would allow me to use the dataset for execution purposes without having to upload it to the context.

I'm wondering if this is possible in the current release. If not, is it something that could be added in a future release?

Thanks!

**Comment by ricklamers** [comment]
You should just be able to add the file to the “workspace/“ folder in the working directory where you run “gptcode” and then just tell the model “use file xyz.zip”
